android/src/main/java/com/brentvatne/exoplayer/ReactExoplayerView.java

```diff
@@ -251,6 +251,8 @@ class ReactExoplayerView extends FrameLayout implements
     private long lastBufferDuration = -1;
     private long lastDuration = -1;
 
+    private boolean srcChanged = false;
```

Comment:
Intention of this variable is to ensure duplicate player stops doesnt come through, whenever we change channel setAnalyticsMeta gets called and then the setSrc gets called and the setAnalyticsMeta function gets called once more hence the introduction of this variable avoids the duplication.